test/common.bash

```diff
@@ -96,7 +96,7 @@ need_squashfs () {
 }
 
 prerequisites_ok () {
-    if [[ -f $CH_TEST_TARDIR/${1}.pq_missing ]]; then
+    if [[ ! -f $CH_TEST_TARDIR/${1}.tar.gz ]]; then
```

Comment:
I think we should leave the `pq_missing` functionality alone. It's OK if this PR doesn't make use of it, but I think it's an important feature. Eventually we may want to hook into it for this.

Also, this will fail if using SquashFS, no?